File: src/main/java/me/kuku/yuq/logic/XiaomiMotionLogic.java
package me.kuku.yuq.logic;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.pojo.Result;

import java.io.IOException;

@AutoBind
public interface XiaomiMotionLogic {
    Result<String> login(String phone, String password) throws IOException;
    String changeStep(String token, int step) throws IOException;
}

File: src/main/kotlin/me/kuku/yuq/logic/XiaomiMotionLogic.kt
package me.kuku.yuq.logic

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.pojo.CommonResult

@AutoBind
interface XiaomiMotionLogic {
    fun login(phone: String, password: String): CommonResult<String>

    fun changeStep(token: String, step: Int): String
}